packages/connect-form/src/components/advanced-options-tabs/general-tab/host-input.tsx

```diff
@@ -32,6 +32,10 @@ const hostActionButtonStyles = css({
   marginTop: spacing[1],
 });
 
+const inputFieldStyles = css({
+  width: '50%',
```

Comment:
Let's share these widths in the form fields in one of the form prs (maybe this one).

Reply:
Let's create a follow-up ticket / PR and merge this one?